tests/e2e/scripts/dts-import.js

```diff
@@ -11,6 +12,59 @@ const {
   engine: { createTransferEngine },
 } = require('@strapi/data-transfer');
 
+function delay(seconds) {
+  return new Promise((resolve) => setTimeout(resolve, seconds * 1000));
+}
+
+async function pollHealthCheck(interval = 1000, timeout = 30000) {
+  const url = `http://127.0.0.1:${process.env.PORT ?? 1337}/_health`;
+  console.log(`Starting to poll: ${url}`);
+
+  let elapsed = 0;
+
+  while (elapsed < timeout) {
+    try {
+      const response = await fetch(url, { method: 'HEAD' });
+      if (response.ok) {
+        console.log('The service is up and running!');
+        return; // Exit if the service is up
+      }
+      // If the response is not okay, throw an error to catch it below
+      throw new Error('Service not ready');
+    } catch (error) {
+      console.log('Waiting for the service to come up...');
+      // Wait for the specified interval before trying again
+      await new Promise((resolve) => setTimeout(resolve, interval));
+      elapsed += interval; // Update the elapsed time
+    }
+  }
+
+  // If we've exited the loop because of the timeout
+  console.error('Timeout reached, service did not become available in time.');
+}
+
+const gitUser = ['-c', 'user.name=Strapi CLI', '-c', 'user.email=test@strapi.io'];
+
+export const resetFiles = async () => {
```

Comment:
im not sure this should be in this file, this feels like it's own script? Or you could just change the name idm. Could do a few things really...

Reply:
oh, yeah, I didn't even pay attention to the filename, I was thinking it was a general utils, good catch 😆

Reply:
I'll also move those three "scripts" to "utils" since that's what they are. It doesn't make sense to have both folders.